bezier/src/cubic_bezier.rs

```diff
@@ -183,6 +183,73 @@ impl CubicBezierSegment {
         find_cubic_bezier_inflection_points(self)
     }
 
+    /// Return local y extremum points or None if this curve is monotone.
+    ///
+    /// This returns the advancements along the curve, not the actual y position.
+    pub fn find_y_extremums(&self) -> UpToTwo<f32> {
```

Comment:
In order to have a coherent API with the quadratic bézier equivalent, this should be called `find_y_inflections`

We also need an equivalent method for x.